### ISSUE TITLE
Implement Component Interface

### ISSUE THREAD

**Issue by podhrmic** [GaloisInc/VERSE-OpenSUT#12]
## Summary
Develop a component interface (depends on #9 ), such that this interface conforms with the relevant standards (SOSA/MOSA, OMS etc.)

## Acceptance Criteria
A set of one or more header/source files that enables a given component to communicate over the messaging bus. The component sends/receives a set of message types, at a specific rate.

## Do
* [ ] message bus interface is defined

**Comment by podhrmic** [comment]
Changing to Someday, as this is a nice-to-have
